Failed to invoke 'SubscribeToOperations' due to an error on the server. InvalidDataException

- **I'm submitting a ...**
  [ *] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
  [ ] question about how to use this project

- **Summary**
Hello, I am getting the error below while trying to create a subscription for monitoring transactions.
```
Error: Error: Failed to invoke 'SubscribeToOperations' due to an error on the server. InvalidDataException: Error binding arguments. Make sure that the types of the provided values match the types of the hub method being invoked.
    at /project-path/node_modules/@tzkt/sdk-events/src/client.ts:292:15
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
[ERROR] 02:31:38 Error: Error: Failed to invoke 'SubscribeToOperations' due to an error on the server. InvalidDataException: Error binding arguments. Make sure that the types of the provided values match the types of the hub method being invoked.
```

- **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)
Here is my snippet code


```
import { EventsService } from '@tzkt/sdk-events';

class Mempool {
  private events: EventsService;

  constructor() {
    this.events = new EventsService({
      url: 'https://api.tzkt.io/v1/events',
      reconnect: true,
    });
  }

  monitor = () => {
    this.events.operations({ types: ['transaction'] }).subscribe({ next: console.log });
  };
}

export const mempool = new Mempool();
````

--- a/packages/ext-taquito/tests/methods.test.ts
+++ b/packages/ext-taquito/tests/methods.test.ts
@@ -67,9 +67,9 @@ describe("request", () => {
     expect(result).toEqual(2664043)
   });
 
-  test('getCounter should equal 45141992', async () => {
-    const result = await extension.getCounter('tz1iG3vqiT95KKSqNQuYnQEXNQk5gXQepM1r', 2664043)
-    expect(Number(result)).toEqual(45141992)
+  test('getCounter should equal 3829', async () => {
+    const result = await extension.getCounter('tz1Nx7hmKnagtzPyWDEba4naJb37Jn6RaP6E', 2664043)
+    expect(Number(result)).toEqual(3829)
   });
 
   test('getBlockTimestamp should equal 2022-08-29T12:06:59Z', async () => {

--- a/packages/sdk-events/package.json
+++ b/packages/sdk-events/package.json
@@ -21,7 +21,7 @@
     "fix:lint": "eslint src --ext .ts --fix",
     "test": "run-s build test:*",
     "test:lint": "eslint src --ext .ts",
-    "test:prettier": "prettier \"src/**/*.ts\" --list-different",
+    "test:prettier": "prettier \"src/**/*.ts\" --list-different --write",
     "test:spelling": "cspell \"{README.md,.github/*.md,src/**/*.ts}\"",
     "check-cli": "run-s test diff-integration-tests check-integration-tests",
     "check-integration-tests": "run-s check-integration-test:*",

--- a/packages/sdk-events/src/client.ts
+++ b/packages/sdk-events/src/client.ts
@@ -189,7 +189,7 @@ export class EventsService {
       const args = sub.params as any;
       // Fix inconsistencies in the subscription interface
       if (sub.method == METHOD.OPERATIONS) {
-        const params = (sub.params as OperationSubscriptionParameters);
+        const params = sub.params as OperationSubscriptionParameters;
         if (params.types && params.types.length > 0) {
           args.types = params.types.join(',');
         }

--- a/packages/sdk-events/src/client.ts
+++ b/packages/sdk-events/src/client.ts
@@ -31,6 +31,7 @@ import {
   AccountSubscriptionParameters,
   BigMapSubscriptionParameters,
   CHANNEL,
+  METHOD,
   channelToMethod,
   Event,
   EventsConfig,
@@ -185,7 +186,15 @@ export class EventsService {
 
   private async invoke(sub: Subscription<ResponseTypes>) {
     if (sub.params) {
-      return await this.connection.invoke(sub.method, sub.params);
+      let args = sub.params as any;
+      // Fix inconsistencies in the subscription interface
+      if (sub.method == METHOD.OPERATIONS) {
+        const params = (sub.params as OperationSubscriptionParameters);
+        if (params.types && params.types.length > 0) {
+          args.types = params.types.join(',');
+        }
+      }
+      return await this.connection.invoke(sub.method, args);
     }
     return await this.connection.invoke(sub.method);
   }

--- a/packages/sdk-events/src/client.ts
+++ b/packages/sdk-events/src/client.ts
@@ -31,12 +31,12 @@ import {
   AccountSubscriptionParameters,
   BigMapSubscriptionParameters,
   CHANNEL,
-  METHOD,
   channelToMethod,
   Event,
   EventsConfig,
   EventType,
   Message,
+  METHOD,
   OperationKind,
   OperationSubscriptionParameters,
   ResponseTypes,
@@ -186,7 +186,7 @@ export class EventsService {
 
   private async invoke(sub: Subscription<ResponseTypes>) {
     if (sub.params) {
-      let args = sub.params as any;
+      const args = sub.params as any;
       // Fix inconsistencies in the subscription interface
       if (sub.method == METHOD.OPERATIONS) {
         const params = (sub.params as OperationSubscriptionParameters);

--- a/packages/sdk-events/src/types.ts
+++ b/packages/sdk-events/src/types.ts
@@ -71,6 +71,7 @@ export interface AccountSubscriptionParameters {
 export interface OperationSubscriptionParameters {
   address?: string;
   types?: Array<OperationKind>;
+  codeHash?: number;
 }
 
 export interface BigMapSubscriptionParameters {

